benefits/oauth/views.py

```diff
@@ -85,71 +85,6 @@ def login(request):
     return result
 
 
-@decorator_from_middleware(FlowUsesClaimsVerificationSessionRequired)
```

Comment:
We lost the middleware here, and it wasn't re-applied in the URL registration. 

How should we handle that?

Reply:
Thanks for catching. Here is a way to do it: https://github.com/cal-itp/benefits/pull/2783/commits/aabf6630b2f14182af28b25e2a9f5c7a17094a2f. Thoughts on it?

Reply:
There's going to be some duplication of the decorators and `{ "hook": ...` kwargs as we implement the various view tickets - #2786, #2787, #2788 - but I think that's ok to start with, and we can clean it up once it's all in.